src/components/victory-axis.jsx

```diff
@@ -193,8 +193,10 @@ class VAxis extends React.Component {
   }
 
   getTickFormat(props) {
-    if (props.tickFormat) {
+    if (props.tickFormat && _.isFunction(props.tickFormat)) {
       return props.tickFormat;
+    } else if (props.tickFormat && _.isArray(props.tickFormat)) {
+      return (x) => props.tickFormat[_.indexOf(this.ticks, x)];
```

Comment:
^ per that comment, add `index` to the call here? https://github.com/FormidableLabs/victory-axis/blob/bugFix/src/components/victory-axis.jsx#L300